bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -242,14 +263,14 @@
 	</channel-type>
 
 	<channel-type id="deviceAFConfigAutoPrefRoomSizeType">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="one">Number:Dimensionless</item-type>
 		<label>Config: Room size</label>
 		<description>Room size (foot sq) for efficient auto mode</description>
 		<state readOnly="true" pattern="%.0f sq ft"/>
 	</channel-type>
 
 	<channel-type id="deviceAFConfigAutoScheduleCountType">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="one">Number:Dimensionless</item-type>
```

Comment:
No unitHint is needed, you could also argue that a non quantitytype is fine

Reply:
I'll install a fresh 4.3 M3 now with the binding added from the store. I'm almost 100% sure the units when the Items are added go straight to being represented for these without the unitHint.

Reply:
You are right: https://github.com/openhab/openhab-webui/blob/7256bc1a3c38d38f16279d7c763d64556ca6e973/bundles/org.openhab.ui/web/src/assets/units.js
Dimensionless defaults to %

Reply:
Yeah I suspect when this binding was done ages ago, it didn't do that so was likely default to 'one', as I remember it being correct before. But hence all the unit hint additions. Which lead's back to the final question at the end of the PR.

Reply:
As last comment have left as is - as this presents correctly the units of "one" without major breakages.